ui/search/nuxeo-results-view.js

```diff
@@ -371,13 +372,8 @@ import './nuxeo-search-results-layout.js';
     }
 
     _onError(e) {
-      this.dispatchEvent(
-        new CustomEvent('notify', {
-          composed: true,
-          bubbles: true,
-          detail: e.detail.error,
-        }),
-      );
+      // XXX
```

Comment:
We should probably add a message on this one.

Reply:
Indeed. I just even even didn't know what to say, so I forgot. 😂  We are firing an error directly instead of an object with a message inside, I just don't know why... 😬

Reply:
Left a message there, please let me know what you think.